Xenon.Tests/SwitchToWindowTests/BaseSwitchToWindowTests.cs

```diff
@@ -31,13 +32,13 @@ public void CanSwitchToNextWindow()
 				CreateInstance( browser )
 					.GoToUrl( "/" )
 					.Click( where => where.TextIs( "Google" ) )
-					.SwitchToWindow( a => a.PageContains( "I'm Feeling Lucky" ) )
+					.SwitchToWindow( a => a.PageContains( "I'm Feeling Lucky" ), WaitArbitrarily )
 					.Assert( a => a.PageContains( "I'm Feeling Lucky" ) )
-					.SwitchToWindow( a => a.PageContains( "Test Page" ) )
+					.SwitchToWindow( a => a.PageContains( "Test Page" ), WaitArbitrarily )
 					.Click( where => where.TextIs( "Yahoo" ) )
-					.SwitchToWindow( a => a.PageContains( "Yahoo" ) )
+					.SwitchToWindow( a => a.PageContains( "Yahoo" ), WaitArbitrarily )
 					.Assert( a => a.PageContains( "Yahoo" ) )
-					.SwitchToWindow( a => a.PageContains( "Test Page" ) )
+					.SwitchToWindow( a => a.PageContains( "Test Page" ), WaitArbitrarily )
```

Comment:
If this arbitrary wait is necessary in all cases when using the Xenon `SwitchToWindow` method, should that not necessarily be part of the method itself?  The point of Xenon being to avoid having to do this sort of thing after all?